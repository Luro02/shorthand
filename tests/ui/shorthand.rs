use shorthand::ShortHand;

#[derive(ShortHand)]
#[shorthand]
struct Example {
    value: usize,
}

#[derive(ShortHand)]
#[shorthand = ""]
struct Example2 {
    value: usize,
}

fn main() {}
