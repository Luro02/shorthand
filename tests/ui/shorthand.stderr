error: unexpected meta-item format `Path`, expected `List`
 --> $DIR/shorthand.rs:4:3
  |
4 | #[shorthand]
  |   ^^^^^^^^^

error: unexpected meta-item format `NameValue`, expected `List`
  --> $DIR/shorthand.rs:10:3
   |
10 | #[shorthand = ""]
   |   ^^^^^^^^^^^^^^
